include/RevMem.h

```diff
@@ -203,17 +203,11 @@ class RevMem {
     return ReadMem( Hart, Addr, sizeof( T ), Target, req, flags );
   }
 
-  ///  RevMem: template LOAD RESERVE memory interface
-  template<typename T>
-  bool LR( unsigned Hart, uint64_t Addr, T* Target, uint8_t aq, uint8_t rl, const MemReq& req, RevFlag flags ) {
-    return LRBase( Hart, Addr, sizeof( T ), Target, aq, rl, req, flags );
-  }
+  ///  RevMem: LOAD RESERVE memory interface
+  void LR( unsigned hart, uint64_t addr, size_t len, void* target, const MemReq& req, RevFlag flags );
```

Comment:
`LR()` can't be `const` because it modifies `LRSC`, a `RevMem` member.
```c++
void RevMem::LR( unsigned hart, uint64_t addr, size_t len, void* target, const MemReq& req, RevFlag flags ) {
  // Create a reservation for this hart, overwriting one if it already exists
  // A reservation maps a hart to an (addr, len) range and is invalidated if any other hart writes to this range
  LRSC.insert_or_assign( hart, std::pair( addr, len ) );
```
The `MemReq` can be passed by `const` reference.

Reply:
Same as the others I was referring to the hart addr and len

Reply:
> Same as the others I was referring to the hart addr and len

What about the hart addr and len? It's an integer address and the len is the size of the load. Since `addr` and `len` are scalar integers passed by value, there is no need to mark them `const` since it just adds to the verbosity of the code without clarifying it.